File: c4/madlib.c
#include <stdio.h>

int main()
{
	char noun[50];
	char verb[50];
	char adjective[50];
	printf("Enter a noun: ");
	gets(noun);
	printf("Enter a verb: ");
	gets(verb);
	printf("Enter an adjective: ");
	gets(adjective);
	printf("Do you %s your %s %s? That sure is disturbing.\n", verb, adjective, noun);
	return 0;
}

File: py4/madlib.py
#!/usr/bin/env python2.7
import sys

def uInputChar(prompt):
    sys.stdout.write(prompt)
    out = str(sys.stdin.readline()).strip()
    return out

def main():
    noun = uInputChar("Enter a noun: ")
    verb = uInputChar("Enter a verb: ")
    adjective = uInputChar("Enter an adjective: ");
    adverb = uInputChar("Enter an adverb: ")
    print("Do you " + verb + " your " + noun + " " + adjective + " " + adverb + "? That is disturbing")
#    sys.stdout.write("Enter a noun

if __name__ == "__main__":
    main()
